src/base/misc/disks.c

```diff
@@ -961,16 +1006,16 @@ static void partition_auto(struct disk *dp)
 
   // Must also set these for build_pi input in setup phase
   dp->start = dp->sectors; // one cylinder for mbr + alignment
+  dp->num_secs += dp->start;
```

Comment:
Is this some fix?
Can it be in a separate patch?

Reply:
Yes, and yes.